tests/grid/grid_transform_02.cc

```diff
@@ -83,7 +83,7 @@ int main ()
             }
 
   // then compute new point locations and output the result
-  GridTools::laplace_transform  (new_points,tria);
+  GridTools::laplace_transform  (new_points,tria, static_cast<const Function<dim, double> *>(NULL), true);
```

Comment:
Sure, but that doesn't explain why you need to add the cast for the third argument or am I missing something?